src/runtime/runtime_unix_heap.go

```diff
@@ -0,0 +1,14 @@
+// +build darwin linux,!baremetal freebsd,!baremetal
```

Comment:
I only now spot this `freebsd,!baremetal` in the code, which doesn't make a lot of sense. The fact that there is `linux,!baremetal` is because baremetal targets pretend to be linux/arm to get the standard library to compile. Such a `!baremetal` exclusion is not necessary for FreeBSD.
But this is not something that needs fixing in this PR.